### PR TITLE
Add swift greeter example

### PR DESCRIPTION
**Pull request by Cosmo**
I've added a greeter example written in swift and also updated the README file.

### CODE REVIEW

File: examples/swift/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## Swift examples
+
+### Permissions
+
+Before running the examples, you have to add execute permissions.
+
+```
+$ chmod +x count.swift
+$ chmod +x greeter.swift
```
Comment (joewalnes) [inline examples/swift/README.md]: You should be able to chmod +x the file and check into git. Anyone who clones will get the right perms.

File: examples/swift/greeter.swift [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env xcrun -sdk macosx swift
+
+import Foundation
+
+while(true){
```
Comment (joewalnes) [inline examples/swift/greeter.swift]: Curious... how does this work? If no messages are incoming, won't it just spin repeatedly printing "Hello !"? Does something block until no data is available?

Comment (Cosmo) [inline examples/swift/greeter.swift]: The magic is in `.availableData`. This method reads the input buffer or blocks if no data is available. Without it, the loop would run non-stop :)

So it's more or less equivalent to `stdin = gets` in ruby.

File: examples/swift/README.md (hunk 1 above)
Comment (Cosmo) [inline examples/swift/README.md]: Oh, I didn't know that, thanks! It looks like, I already had the files checked into github with +x.
I'll remove the permissions part.
You never stop learning :+1:

**Comment by Cosmo** [comment]
Thanks @joewalnes for letting me know, that git handles file modes. :heart: 
It looks like I've commited the example files with +x mode already.

**Comment by joewalnes** [comment]
Thanks - merged!

**Pull request merged**